Launcher/config.json

```diff
@@ -142,6 +142,15 @@
   "binanceus-api-url": "https://api.binance.us",
   "binanceus-websocket-url": "wss://stream.binance.us:9443/ws",
 
+
+  // bybit configuration
+  "bybit-api-secret": "",
+  "bybit-api-key": "",
+  "bybit-api-url": "https://api.bybit.com",
+  "bybit-websocket-url": "wss://stream.bybit.com",
+  "bybit-orderbook-depth": 50,
+  "bybit-futures-oderbook-depth": 50,
```

Comment:
Suggest removing the `depth` configurations for now 👍 and default to the best possible